bundles/org.openhab.binding.netatmo/src/main/resources/OH-INF/update/instructions.xml

```diff
@@ -14,6 +14,12 @@
 			</update-channel>
 		</instruction-set>
 
+		<instruction-set targetVersion="2">
```

Comment:
You also need increase the thing property to 2.

Reply:
It is done in `ModuleType.java`, remember that Netatmo binding does not use xml based thing type definition but  dynamical.

Reply:
Thanks, i see now.